woocommerce/google-listings-and-ads
# Adjust click event tracking when connecting, disconnecting, and opening billing setup for Google Ads account

### Changes proposed in this Pull Request:

PT: pcTzPl-2iK-p2

The PR adds/adjusts three tracking events for for Google Ads account:

- Add `context` and `step` event properties to the click event tracking in `ConnectAds`.
- Add the `gla_ads_account_disconnect_button_click` event with `context` and `step` event properties to the click event tracking in `DisconnectAccount`.
- Adjust `context` and add `step` event properties to the click event tracking in `BillingSetupCard`.
- Extra change: Remove the unused component `GoogleAdsAccountSection`.

### Detailed test instructions:

1. Open the Console tab in the browser DevTool, and run `localStorage.setItem( 'debug', 'wc-admin:*' )` to enable tracking debugging mode.
2. Disconnect Google Ads account if there is a connected one.
3. Go to step 1 of the **Extension onboarding**.
4. Connect an existing Google Ads account.
5. Check if the `gla_ads_account_connect_button_click` event is sent with `id`, `context` and `step`.
6. Disconnect the Google Ads account.
7. Check if the `gla_ads_account_disconnect_button_click` event is sent with `context` and `step`.
8. Create a new Google Ads account but don't set up billing, and proceed to step 4 of the onboarding flow.
9. Open the campaign creation setup.
10. Click on the "Set up billing" button to see if it can open the billing setup page in a pop-up window.
11. Check if the `gla_ads_set_up_billing_click` event is sent with `link_id`, `href`, `context` and `step`.
12. Click on the "click here instead" link to see if it can open the billing setup page in a new browser tab.
13. Check if the `gla_ads_set_up_billing_click` event is sent with `link_id`, `href`, `context` and `step`.

### Changelog entry

> Tweak - Adjust click event tracking when connecting, disconnecting, and opening billing setup for Google Ads account.
